contracts/LoanFactory.sol

```diff
@@ -1,386 +1,83 @@
 pragma solidity 0.4.24;
 
-/**
- * @title Ownable
- * @dev The Ownable contract has an owner address, and provides basic authorization control
- * functions, this simplifies the implementation of "user permissions".
- */
-contract Ownable {
+import "./Loan.sol";
+
+contract LoanFactory {
     address public owner;
+    address public worker;
+    address[] public loans;
+    uint256 public loanId;
+    mapping(bytes32 => uint256) public marginLeadTime;
+    mapping(bytes32 => uint256) public matureLeadTime;
+    mapping(bytes32 => uint256) public interestLeadTime;
 
-    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+    event NewLoan(address indexed loan, uint256 loanId);
 
-    /**
-     * @dev The Ownable constructor sets the original `owner` of the contract to the sender
-     * account.
-     */
-    constructor () public {
-        owner = msg.sender;
-    }
-
-    /**
-     * @dev Throws if called by any account other than the owner.
-     */
     modifier onlyOwner() {
         require(msg.sender == owner);
         _;
     }
```

Comment:
duplicate modifier (also exists in `Loan`)

Reply:
loan does not inherits from loanFactory and the function is also different in loan